types/jest-expect-message/index.d.ts

```diff
@@ -5,17 +5,18 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.3
 
-/// <reference types="jest"/>
+import { Matchers } from 'expect';
 
-declare namespace jest {
-    interface Expect {
+// tslint:disable-next-line: no-single-declare-module
+declare module 'expect' {
+    interface BaseExpect {
         /**
          * The `expect` function is used every time you want to test a value.
          * You will rarely call `expect` by itself.
          *
          * @param actual The value to apply matchers against.
          * @param message Clarification message
          */
-        <T = any>(actual: T, message: string): JestMatchers<T>;
+        (actual: any, message: string): Matchers<void>;
```

Comment:
Just to be sure. Removal of the generic could possibly mess up other matchers. For example, does it work with `.resolves` chainer?